File: sentimental-mario-less/mario.py
from cs50 import get_int
# Ask user for input if input less than 0 or more than 8 repet ask
while True:
    input = get_int("Input height of pyramid: ")
    # Check if input between 1 and 8
    if input >= 1 and input <= 8:
        break
# Doing loop and draw stairs
for i in range(1, input + 1):
    print(" " * (input - i) + "#" * i)

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Input height of pyramid: ");
    }
    while (h < 1 || h > 8);
    // Count for columns
    for (int i = 1; i < h + 1; i++)
    {
        // Count for print spaces
        for (int z = h; z > i; z--)
        {
            printf(" ");
        }
        //Count to print hash
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}